client/src/components/payment/refund-summary.vue

```diff
@@ -36,26 +36,22 @@
 <script lang="ts">
 import { Component, Prop, Vue } from 'vue-property-decorator'
 
-const _flatten = require('lodash/flatten')
-
 @Component({})
 export default class RefundSummary extends Vue {
   @Prop({ default: () => [] })
   readonly payments: any[]
 
   /** The array of line items in all completed SBC payments. */
   private get lineItems (): any[] {
-    const lineItems = _flatten(
-      this.payments.map(p => (p.sbcPayment.statusCode === 'COMPLETED') ? p.sbcPayment.lineItems : [])
-    )
+    const arrays = this.payments.map(p => (p.sbcPayment.statusCode === 'COMPLETED') ? p.sbcPayment.lineItems : [])
+    const lineItems = [].concat(...arrays)
```

Comment:
ES6 has a new array method `flat` but, according to http://caniuse.com, not all browser suport it, while `concat` is OK.
While I was there, I also confirmed that `reduce` is also well supported.